toolchain-m68k

```diff
@@ -241,6 +241,14 @@ def build():
                               path.join('{host}', 'bin'),
                               environ['PATH']])
 
+  """
+  When we have a working compiler in our path, we can also check if the
+  required headers/libraries are present.
+  """
+
+  require_header('<python2.7/python.h>')
+  require_header('<ncurses.h>', symbol = 'NCURSES_VERSION_MAJOR', value = 5)
```

Comment:
This is just a minor detail... but I guess we'd like to always ask about system includes (embedded in `<>`) so it can be moved to `require_header`.

It'd be nice to have a human readable failure message like "*libncurses-dev version 5.x missing*".